medusa/common.py

```diff
@@ -545,7 +529,7 @@ def sceneQualityFromName(name, quality):
             elif re.search(r'web(-| |\.)?(rip|mux)', name):
                 rel_type = ' WEBRip'
 
-        return rel_type + codec
+        return rel_type
```

Comment:
@medariox @duramato do we need codec here too?

If so we add add in the call to sceneQuality, not here.